docs/src/running-validator/validator-reqs.md

```diff
@@ -63,7 +63,13 @@ Solana.
 
 In fact, the mainnet-beta validators operated by the team are currently
 (Mar. 2021) run on GCE `n2-standard-32` (32 vCPUs, 128 GB memory) instances with
-2048 GB SSD for operational convenience.
+2048 GB SSD for operational convenience. If you prefer AWS then (based on tests ran by vendor 
+later on May 2023) `r6a.8xlarge` (32 vCPU, 256 GB memory) instances work well with 
+three EBS gp3 volumes for better performance:
+
+- Root: EBS gp3, 100GB, 3K IOPS, 125 MB/s throughput;
+- Accounts volume: EBS gp3, 1TB, 5K IOPS, 700 MB/s throughput; 
+- Data volume: EBS gp3, 3TB, 10K IOPS, 700 MB/s throughput).
```

Comment:
```suggestion
- Root: EBS gp3, 100GB, 3K IOPS, 125 MB/s throughput
- Accounts volume: EBS gp3, 1TB, 5K IOPS, 700 MB/s throughput 
- Data volume: EBS gp3, 3TB, 10K IOPS, 700 MB/s throughput
```

Reply:
No, they are not quoted, but without them it will be hard for the operators to set up a node. I'll summarize this into a brief form. The phrase still has phrasing to identify that this is a point-in-time data.